src/btree/bt_sync_obsolete.c

```diff
@@ -822,18 +822,18 @@ __wt_checkpoint_cleanup_create(WT_SESSION_IMPL *session, const char *cfg[])
 
     WT_RET(__wt_cond_alloc(session, "checkpoint cleanup", &conn->cc_cleanup.cond));
 
-    WT_RET(__wt_thread_create(session, &conn->cc_cleanup.tid, __checkpoint_cleanup, session));
+    WT_RET(__wt_thread_create(session, &conn->cc_cleanup.tid, __obsolete_cleanup, session));
     conn->cc_cleanup.tid_set = true;
 
     return (0);
 }
 
 /*
- * __wt_checkpoint_cleanup_destroy --
+ * __wt_obsolete_cleanup_destroy --
  *     Destroy the checkpoint cleanup thread.
```

Comment:
And here. I think search the whole file for the word `checkpoint` (or just `check`)